hw/ip/otbn/data/otbn.hjson

```diff
@@ -253,7 +253,22 @@
           desc: "Set on any ECC error in a register file"
         }
       ]
-    }
+    } // register : fatal_alert_cause
+    { name: "INSN_CNT",
+      desc: "Instruction Counter",
+      swaccess: "ro",
+      hwaccess: "hwo",
+      fields: [
+        { bits: "31:0",
+          name: "insn_cnt",
+          desc: '''
+            The number of instructions executed in the current or last OTBN run. 
+            Saturetes at 2^32-1. The counter is reset to zero when a new operation
+            is started.
```

Comment:
Let's clarify the behavior in case of errors (see also other comment). Maybe add a sentence like.

"Instructions triggering an error do not count as being executed."